c10/mobile/CPUCachingAllocator.h

```diff
@@ -10,6 +10,39 @@
 #include <c10/util/SmallVector.h>
 #include <c10/util/flat_hash_map.h>
 
+/*
+ * CPUCachingAllocator:
+ * Why?
+ *    It has been observed that on mobile some platforms, such as pixel 3, returns
```

Comment:
nit: did you mean: "It has been observed that some mobile platforms, such as pixel 3", return ..."